ccmlib/repository.py

```diff
@@ -14,17 +14,19 @@
 from distutils.version import LooseVersion
 
 from six import print_
-from six.moves import urllib
 
-from ccmlib.common import (ArgumentError, CCMError, get_default_path,
-                           platform_binary, rmdirs, validate_install_dir,
-                           assert_jdk_valid_for_cassandra_version, get_version_from_build)
+from ccmlib.common import (ArgumentError, CCMError,
+                           assert_jdk_valid_for_cassandra_version,
+                           get_default_path, get_version_from_build,
+                           platform_binary, rmdirs, validate_install_dir)
+from six.moves import urllib
 
 DSE_ARCHIVE = "http://downloads.datastax.com/enterprise/dse-%s-bin.tar.gz"
 OPSC_ARCHIVE = "http://downloads.datastax.com/community/opscenter-%s.tar.gz"
 ARCHIVE = "http://archive.apache.org/dist/cassandra"
 GIT_REPO = "http://git-wip-us.apache.org/repos/asf/cassandra.git"
 GITHUB_TAGS = "https://api.github.com/repos/apache/cassandra/git/refs/tags"
+LOCAL_GIT_REPO = os.environ.get('LOCAL_GIT_REPO', None)
```

Comment:
Can you comment an example of what this should look like? It's unclear to me how this works.